### PR TITLE
Removed center from points

### PR DESCRIPTION
**Pull request by SukantPal**
#6562 made a regression in the stroke. The "points" array is used to do a stroke, and adding the center there add a line from the center to circumference. This reverts that and transforms the center in `triangulate` instead.

### CODE REVIEW

**Comment by SukantPal** [comment]
https://jsfiddle.net/ShukantPal/u6p5rvs7/3/

**Comment by SukantPal** [comment]
https://jsfiddle.net/ShukantPal/uq5f4k8o/ is this update.

File: packages/graphics/src/GraphicsData.ts [hunk 1]
```diff
@@ -17,6 +17,7 @@ export class GraphicsData
     type: SHAPES;
     points: number[];
     holes: Array<GraphicsData>;
+
```
Comment (bigtimebuddy) [inline packages/graphics/src/GraphicsData.ts:20]: Can you remove this white space?

**Comment by bigtimebuddy** [comment]
Thank you!

**Pull request merged**